x-pack/test_serverless/functional/test_suites/security/ftr/landing_page.ts

```diff
@@ -14,11 +14,7 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
 
   describe('landing page', function () {
     before(async () => {
-      await svlCommonPage.login();
-    });
-
-    after(async () => {
-      await svlCommonPage.forceLogout();
+      await svlCommonPage.loginWithPrivilegedRole();
```

Comment:
Same question here. A privileged role might not be necessary.

Reply:
https://github.com/elastic/kibana/pull/188128/commits/084779204c7885739b195b58ce4b448a41d9a6c2